src/bundles/Collaborate/components/Code/Code.js

```diff
@@ -61,7 +76,8 @@ class Code extends React.Component {
                                                 <a rel="external" target="_blank" href={item.url}
                                                    onClick={(e) => e.stopPropagation()}>{item.name}</a>
```

Comment:
React has an "interesting" way of handling events such that they propagate up the DOM ie. this link is inside another thing that can be clicked on so they both get click events.